src/vorta/views/repo_add_dialog.py

```diff
@@ -124,21 +165,44 @@ def validate(self):
             self._set_status(self.tr('This repo has already been added.'))
             return False
 
-        if self.__class__ == AddRepoWindow:
-            if self.values['encryption'] != 'none':
-                if len(self.values['password']) < 8:
-                    self._set_status(self.tr('Please use a longer passphrase.'))
-                    return False
-
         return True
 
+    def password_listener(self):
```

Comment:
Again, we could consider simplifying this PR by only validating the password on submission, not in real time.

Reply:
The code needed to validate passwords is just two lines, not complex at all.
```
        self.passwordLineEdit.textChanged.connect(self.password_listener)
        self.confirmLineEdit.textChanged.connect(self.password_listener)
```

Reply:
Fine. More interactive.